src/libsystemd-network/sd-dhcp-lease.c

```diff
@@ -780,6 +843,37 @@ int sd_dhcp_lease_load(sd_dhcp_lease **ret, const char *lease_file) {
                            "ROUTES", &routes,
                            "CLIENTID", &client_id_hex,
                            "VENDOR_SPECIFIC", &vendor_specific_hex,
+                           "OPTION_224", &options[DHCP_OPTION_PRIVATE_0 - DHCP_OPTION_PRIVATE_START],
```

Comment:
Hmm, actually it doesn't. I get no warning about `&options[31]`, but it does complain if I write `options[31]`. I guess it's treating `&options[31]` the same way as it would `options + 31`.